lib/graphql/relay/base_connection.rb

```diff
@@ -53,17 +53,18 @@ def register_connection_implementation(nodes_class, connection_class)
         deprecate(:connection_for_items, :connection_for_nodes, 2016, 9)
       end
 
-      attr_reader :nodes, :arguments, :max_page_size, :parent
+      attr_reader :nodes, :arguments, :max_page_size, :parent, :field
 
       # Make a connection, wrapping `nodes`
       # @param [Object] The collection of nodes
       # @param Query arguments
       # @param max_page_size [Int] The maximum number of results to return
       # @param parent [Object] The object which this collection belongs to
-      def initialize(nodes, arguments, max_page_size: nil, parent: nil)
+      def initialize(nodes, arguments, field: field, max_page_size: nil, parent: nil)
```

Comment:
I don't recognize this syntax, `field: field,`, is it meant to be a required keyword, `field:,` or does this default it to `nil`, since it calls the `attr_reader`?